ext/RMagick/rmutil.c

```diff
@@ -1320,7 +1342,7 @@ rm_exif_by_entry(Image *image)
     while (property)
     {
         // ignore properties that don't start with "exif:"
-        property_l = strlen(property);
+        property_l = rm_strnlen_s(property, MaxTextExtent);
```

Comment:
@Watson1978 some places use `strlen` and others `strnlen`. What is the difference?

Reply:
strlen() has been called on some apparently fixed strings. So, I didn't change them because there is no problem.